pingpong/lecture_video_processing.py

```
@@ -458,17 +565,442 @@ async def _await_with_run_lease_heartbeat(
+            )
+            openai_client = await get_openai_client_by_class_id(session, class_id)
+
+        logger.info(
+            "Lecture video manifest loading Gemini credential. "
+            "run_id=%s lecture_video_id=%s class_id=%s",
+            run_id,
+            lecture_video_id,
+            class_id,
+        )
+        gemini_api_key = await _get_gemini_api_key(class_id)
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)